api/tests/Feature/UserTest.php

```diff
@@ -683,10 +683,16 @@ public function testFilterByCandidateExpiryDate(): void
 
     public function testFilterByLanguageAbility(): void
     {
-        User::factory()->count(5)->create([
-            'looking_for_english' => false,
-            'looking_for_french' => false,
-            'looking_for_bilingual' => false,
+        $noLanguageUser = User::factory()->create();
+        $noLanguageUser->looking_for_english = false;
```

Comment:
To set intial values, pass an object with key-value pairs to `create`, like on line 693.